app/src/main/java/org/oppia/android/app/player/state/itemviewmodel/SelectionInteractionViewModel.kt

```diff
@@ -148,6 +153,18 @@ class SelectionInteractionViewModel private constructor(
     }
   }
 
+  private fun updateSelectionText() {
```

Comment:
This function also needs to update the enabled states of each checkbox based on whether more options can be selected (it needs to disable unselected checkboxes when the limit is reached, and enable all checkboxes if the limit is not reached).

Reply:
Okay

Reply:
This doesn't seem done yet--am I missing something?

Reply:
done.

Reply:
Not done yet--all that's being changed is the selection item next, not the actual checkbox enabled/disabled states. I left another comment in SelectionInteractionContentViewModel as well, PTAL.

Reply:
I think ben this function is only for the text as I mentioned above that this viewmodel's Xml is only implementing the textView and the radioButton is implemented by `multiple_choice_interaction_items.xml` which is part of `SelectionInteractionContentViewModel` and Ben now the code is working fine Ig.